lib/helpers/dotnetutils.test.js

```diff
@@ -0,0 +1,413 @@
+import { expect, test } from "@jest/globals";
```

Comment:
I hope we have covered many test cases here. Please review and suggest if there is anything missing.